cmd/influxd/config.go

```diff
@@ -312,6 +313,11 @@ func (d *Duration) UnmarshalText(text []byte) error {
 	return nil
 }
 
+// WriteConfigFile writes the config to the specified writer
+func WriteConfigFile(c *Config, w io.Writer) error {
```

Comment:
@otoolep yeah, it was almost flip a coin for me which way to go.  I do agree I think it's a method, not a function.  updated.